setup.py

```diff
@@ -5,13 +5,15 @@
 from __future__ import unicode_literals, print_function, division, absolute_import
 from setuptools import setup, find_packages
 
+# Please make sure to cap all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
 
 general_requirements = [
     'Biryani[datetimeconv] >= 0.10.8',
     'dpath == 1.4.0',
     'enum34 >= 1.1.6',
-    'future',
-    'nose',  # For openfisca-run-test
+    'future < 1.0.0',
```

Comment:
I doubt this protect us from anything. Pre-`1.0`, there can be breaking changes without bumping the first digit...